# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/mediasink/AudioSink.h
# Commit: af366e7b6233
# Full Hash: af366e7b62333a382d506ba103fc414c451c5fd3
# Author: Paul Adenot <paul@paul.cx>
# Date: 2022-02-12 09:47:43
# Regressor Bug: 1754006
# File Overlap Count: 1
# Description:
#   Bug 1754006 - Increase the processed media queue size and threshold in AudioSink. r=alwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138028
# ==============================================================================

diff -r b4ab5515b401 -r af366e7b6233 dom/media/mediasink/AudioSink.h
--- a/dom/media/mediasink/AudioSink.h	Fri Feb 11 16:34:24 2022 +0000
+++ b/dom/media/mediasink/AudioSink.h	Fri Feb 11 16:58:51 2022 +0000
@@ -129,7 +129,7 @@
       AlignedAudioBuffer&& aBuffer, AudioData* aReference);
   // Add data to the processsed queue return the number of frames added.
   uint32_t PushProcessedAudio(AudioData* aData);
-  uint32_t AudioQueuedInRingBufferUs() const;
+  uint32_t AudioQueuedInRingBufferMS() const;
   uint32_t SampleToFrame(uint32_t aSamples) const;
   UniquePtr<AudioConverter> mConverter;
   UniquePtr<SPSCQueue<AudioDataValue>> mProcessedSPSCQueue;
@@ -153,6 +153,7 @@
 
   Atomic<bool> mProcessedQueueFinished;
   MediaQueue<AudioData>& mAudioQueue;
+  const float mProcessedQueueThresholdMS;
 };
 
 }  // namespace mozilla